cpp/src/tsne/tsne_test.cu

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
Should this file be removed / moved to tests?

Reply:
Oops